core/src/main/java/io/hyperfoil/core/util/LongFastUUID.java

```diff
@@ -0,0 +1,35 @@
+package io.hyperfoil.core.util;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Generate UUID based on the following match: [0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}
+ */
+public class LongFastUUID {
+   private LongFastUUID() {
+
+   }
+
+   private static final Long LONG_INSTANCE = Long.MAX_VALUE;
+   private static final Random RANDOM = ThreadLocalRandom.current();
+   private static Method fastUUID;
+   static {
+      try {
+         fastUUID = Long.class.getDeclaredMethod("fastUUID", long.class, long.class);
+      } catch (NoSuchMethodException e) {
+         throw new IllegalArgumentException(e);
+      }
+      fastUUID.setAccessible(true);
+   }
+
+   public static String randomUUID() {
+      try {
+         return (String) fastUUID.invoke(LONG_INSTANCE, RANDOM.nextLong(), RANDOM.nextLong());
```

Comment:
The correct pattern to invoking static methods via reflection is using `null` for the instance parameter.